frontend/public/components/secrets/create-secret/KeyValueEntryForm.tsx

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import * as React from 'react';
+import { withTranslation } from 'react-i18next';
+import { AsyncComponent } from '../../utils/async';
+import { WithT } from 'i18next';
+
+export const DroppableFileInput = (props) => (
```

Comment:
No need to export this. 

Also, we should leave a TODO comment on this component to create a shared LazyDroppableFileInput component. This lazy-loaded DroppableFileInput pattern is repeated in multiple places in our codebase.